JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/derivation/DerivationSparql.java

```diff
@@ -190,7 +203,7 @@ public DerivationSparql(StoreClientInterface storeClient, String derivationInsta
 	 * @return
 	 */
 	List<String> unifiedBulkCreateDerivations(List<List<String>> entitiesList, List<String> agentIRIList,
```

Comment:
Also I thought we want to get rid of methods that takes in both agent URL and IRI?

Reply:
I will do this in a separate PR.